scripts/test_caddy_supabase_routes.py

```
@@ -0,0 +1,293 @@
+
+failures = []
+
+
+def fail(msg):
+    failures.append(msg)
+
+
+def strip_comments(text):
+    """Drop whole-line comments so a commented-out directive cannot satisfy a
+    check, and a comment mentioning a prefix cannot trip one."""
+    return "\n".join(l for l in text.splitlines() if not l.strip().startswith("#"))
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)